tfbpmodeling/lasso_modeling.py

```diff
@@ -1474,92 +1474,140 @@ def final_model(self) -> list[str]:
         return df["selected"].tolist()
 
 
-def evaluate_interactor_significance(
+def evaluate_interactor_significance_linear(
```

Comment:
This entire function should not have been changed from the original evaluate_interactor_significance aside from the name